lib/burnside/bridge.cc

```diff
@@ -94,6 +94,10 @@ class FirConverter : public AbstractConverter {
     return createI1LogicalExpression(
         loc, *builder, *expr, localSymbols, defaults, intrinsics);
   }
+  M::Value *createTemporary(M::Location loc, const Se::Symbol &sym) {
+    return ::createTemporary(loc, *builder, localSymbols,
```

Comment:
Is this function being called not in a namespace?

Reply:
I saw you have already changed this to `Br::` thanks, that is clearer than `::`.